commands.go

```diff
@@ -537,6 +538,23 @@ func (c cmdable) Command(ctx context.Context) *CommandsInfoCmd {
 	return cmd
 }
 
+func (c cmdable) CommandList(ctx context.Context, opts *FilterBy) *StringSliceCmd {
```

Comment:
Suggest renaming: opts -> opt or filter, opts generally stands for [ ]option

Reply:
@monkey92t Sure